strawberryfields/apps/plot.py

```diff
@@ -125,7 +125,7 @@ def graph(g: nx.Graph, s: Optional[list] = None, plot_size: int = 500):  # pragm
     **Example usage:**
 
     >>> graph = nx.complete_graph(10)
-    >>> fig = graph(graph, [0, 1, 2, 3])
```

Comment:
@ixfoduap @soranjh One thing to note is that the codeblocks looked a bit odd - we had for example
```python
>>> fig = graph(graph, [0, 1, 2, 3])
```
for plotting. It looks clearer when we specify the module:
```python
>>> fig = plot.graph(graph, [0, 1, 2, 3])
```
Although we don't normally do this, I think it's fair to make an exception here.